src/sock.c

```diff
@@ -234,6 +234,25 @@ struct dnsquery_resourcerecord
 	struct dnsquery_srvrdata rdata;
 };
 
+void netbuf_add_32bitnum(unsigned char *, int, int *, unsigned int);
```

Comment:
It just removes warnings when -Wmissing-prototype is used with gcc. I try to remove most of them in order to see only the real problems I have with libstrophe on my arch [OpenBSD/loongson](http://www.openbsd.org/loongson.html). As you can see it is not important, just convenient.